### PR TITLE
Withdraw payment + docs

### PR DESCRIPTION
**Pull request by RodrigoAD**
Gauntlet Command to withdraw payments for Node Ops and Documentation

### CODE REVIEW

**Review by aalu1418 — COMMENTED** [review]
Ideally I would like to see a test case with this not sure how easy that is?

Or atleast confirmation that this has been tested and verified that it works properly

**Comment by RodrigoAD** [comment]
Ideally we would have Gauntlet tests, but for now it can only be manually tested. I did it myself and works as expected. The tx signer needs to be the owner of the payee address (token derived account of the signer)

File: docs/gauntlet/withdraw.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Withdraw Payments using Gauntlet
+
+When Node Operators report data to a feed, they are compensated with LINK thats owned by the feed. Node operators can then withdraw the funds owed to them via gauntlet using this section below
+
+## Sample Command Usage
+
+Make sure to set up gauntlet using the steps described in [Gauntlet Setup](gauntlet-setup.md) before attempting to run the following command
+
+```bash
+yarn gauntlet ocr2:withdraw_payment --network=mainnet --recipient=<VALID_LINK_RECIPIENT_ADDRESS> <FEED_ADDRESS>
```
Comment (aalu1418) [inline docs/gauntlet/withdraw.md]: What does `VALID_LINK_RECIPIENT_ADDRESS` mean? I think it means that it is the associated token account...is that correct? 

Would be good to also add something like `VALID_LINK_RECIPIENT_ADDRESS` = the same account address used in setting the payee config (should be the associated account for the LINK token specific to the owner public key)

**Review by aalu1418 — CHANGES_REQUESTED** [review]

File: docs/gauntlet/withdraw.md (hunk 1 above)
Comment (RodrigoAD) [reply docs/gauntlet/withdraw.md]: Updated it @aalu1418, lmk if it's more clear now

**Review by aalu1418 — APPROVED** [review]
lgtm!

**Pull request merged**